tensorboard/webapp/metrics/views/card_renderer/scalar_card_component.ng.html

```diff
@@ -119,7 +119,8 @@
   <ng-template
     #tooltip
     let-tooltipData="data"
-    let-cursorLoc="cursorLocationInDataCoord"
+    let-cursorLocationInDataCoord="cursorLocationInDataCoord"
```

Comment:
Whatever names you choose line_chart_interactive_view.ts, please mirror here.

Reply:
These names all look the same to me? Am I missing something? (of note, they were not the same before and I renamed them to be the same).

Reply:
There was another comment I left about naming inconsistency, which does not seem to have been addressed. Do you mind taking a look at that one and then revisiting this one? I assume this comment doesn't make sense to you because you missed the other comment.